Examples/Headless/GettingStarted/Program.cs

```diff
@@ -14,13 +14,15 @@ static void Main()
 			AppCoreMethods.ulEnablePlatformFontLoader();
 
 			// Set filesystem (Ultralight requires "resources/icudt67l.dat")
-			AppCoreMethods.ulEnablePlatformFileSystem(Path.GetDirectoryName(typeof(Program).Assembly.Location));
+			var path = Path.GetDirectoryName(typeof(Program).Assembly.Location);
+			AppCoreMethods.ulEnablePlatformFileSystem(path);
 
 			// Create Renderer
-			Renderer renderer = ULPlatform.CreateRenderer();
+			var cfg = new ULConfig();
+			Renderer renderer = ULPlatform.CreateRenderer(cfg);
```

Comment:
That is... Api design *that is not implemented yet* 😅

Reply:
What? I have tested it offline and it works, no idea what you mean.